scrapers/RealityKings.yml

```diff
@@ -66,19 +66,29 @@ xPathScrapers:
             with: https://www.realitykings.com
 
   sceneScraper:
+    common:
+      $section: //img[@alt=""][contains(@src,"poster")]/../../.. #img>div>div>section
     scene:
-      Title: //h2[contains(@class, "hzrLcs")]/text()
+      Title: $section//h1/text()|$section//h2/text()
       Date:
-        selector: //div[contains(@class, "jtRRdJ")]/text()
+        selector: //div[text()="Release Date:"]/../text()
```

Comment:
This selector does not appear to work on realitykings.com. 
Example: https://www.realitykings.com/scene/5258/so-seduce-me this scene is on realitykings.com does not have a div with "Realease Date". This same scene can be found here https://www.momsbangteens.com/scene/5258/so-seduce-me. It does have a div with a "Release Date". This is one example of differences between RealityKings.yml and RealityKingsOL.yml.

Perhaps use this for the date selector in RealityKings.yml:

selector: //h2/preceding-sibling::div/text()